.circleci/config.base.yml

```diff
@@ -103,8 +122,21 @@ jobs:
               - equal: [*windows-e2e-executor, << parameters.os >>]
           steps:
             - checkout
-            - run: nvm install 12.22.7
-            - run: nvm alias default 12.22.7
+            - run:
+                name: Install Node 18
+                command: |
+                  nvm version
+                  nvm install 18.15.0
+                  nvm use 18.15.0
+                  nvm ls
```

Comment:
Is this here as a debug step?

Reply:
Yes it is. It lists all the installed Node versions and highlights the **default**. We can get rid of it but I don't see any risk in keeping it.